.pylintrc

```diff
@@ -17,6 +17,8 @@ ignore-patterns=.git
 # pygtk.require().
 #init-hook=
 
+init-hook='import sys; sys.path.append(".")'
```

Comment:
Isn't doing a `make develop` a better option here?

Reply:
Yes, I considered using make develop here too. But I would like to remove the "Makefile dependency" really soon. Besides that, make develop is also installing optional plugins that will be removed from this repo really soon (I hope).

But I guess that makes sense to install in develop mode, so I will drop this commit and use " python3 setup develop" during the steps in the workflow. Thank you.